js/siteorigin-panels.js

```diff
@@ -3469,13 +3469,23 @@ module.exports = Backbone.View.extend( {
         // Make sure we actually need to copy content.
         if( panelsOptions.copy_content && this.attachedToEditor && this.$el.is(':visible')) {
```

Comment:
If you spot the issue, feel free to fix up `attachedToEditor` while you're at it ;)